common/djangoapps/course_modes/views.py

```diff
@@ -226,3 +228,31 @@ def _get_requested_mode(self, request_dict):
             return 'honor'
         else:
             return None
+
+
+@ensure_csrf_cookie
+@require_level('staff')
```

Comment:
Here's the URL: https://github.com/edx/edx-platform/wiki/Shopping-Cart#paid-course-enrollments

We tried to use bok-choy to automate clicks on the admin site, but we dont have a user with sufficient rights. Here's what the document ^^ says:

"To configure a course to be a Paid Course Enrollment, a system administrator will need to go to the Django Admin website that is at the /admin URL."